deltachat-ffi/deltachat.h

```diff
@@ -3701,6 +3701,9 @@ char*             dc_msg_get_webxdc_blob      (const dc_msg_t* msg, const char*
  *   To get the file, use dc_msg_get_webxdc_blob().
  *   App icons should should be square,
  *   the implementations will add round corners etc. as needed.
+ * - summary: short string describing the state of the app,
+ *   sth. as "2 votes", "Highscore: 123",
+ *   can be changed by the apps and defaults to an empty string.
```

Comment:
wouldn't it make also sense to have multiline text for a leader board or poll-results?

Reply:
that would raise much more issues for comparable few additional effects:
- rendering gets complicated as messages bubbles may grow/shrink vertically
- showing information that are not aggregated easier raise races

the summary is just meant to show that "something has changed" and it is worth opening the app.